CIP-0021/CIP-0021.md

```diff
@@ -71,6 +79,20 @@ It is allowed to arbitrarily combine other supported certificate types, but all
 
 Since withdrawals are represented as a map of reward accounts, withdrawals also need to be sorted in accordance with the specified canonical CBOR format. A transaction must not contain duplicate withdrawals. All withdrawals included in a transaction must have the same type of stake credential i.e. either 0 - `addr_keyhash` or 1 - `scripthash`. The stake credential type must also be consistent with the type used for certificates.
 
+**Auxiliary data**
+
+Auxiliary must be encoded in their "tuple" format i.e.:
```

Comment:
```suggestion
HW wallets do not serialize auxiliary data because of their complex structure --- they only include the given auxiliary data hash in the transaction body. The only exception is Catalyst voting registration because it requires a signature computed by the hw wallet. 

In this exceptional case, auxiliary data must be encoded in their "tuple" format:
```

Reply:
https://github.com/vacuumlabs/CIPs/pull/4/commits/707b4f37e97c2b4287930bc49efa53ec864c9ac7